_developer-documentation/index.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+title: Developer documentation
+nav_order: 1
+has_children: false
+has_toc: false
+nav_exclude: true
+---
+
+# Developer documentation
+
+We welcome your contributions to the OpenSearch Project. Here are some helpful links to explore the OpenSearch repositories and learn how to contribute:
+
+- [OpenSearch Project GitHub repo](https://github.com/opensearch-project/)
+- [Javadoc documentation](https://opensearch.org/javadocs/)
+- [Getting started as an OpenSearch contributor](https://github.com/opensearch-project/.github/blob/main/ONBOARDING.md)
+- [OpenSearch release schedule and maintenance policy](https://opensearch.org/releases.html)
+- [OpenSearch Project roadmap](https://github.com/orgs/opensearch-project/projects/1)
+- [OpenSearch Community Forum](https://forum.opensearch.org/)
+
+## What's new
+
+New in version 2.8, OpenSearch introduces _extensions_---an easier to develop and more secure alternative to plugins, which simplifies creating custom functionality for OpenSearch. To learn more about building extensions using _OpenSearch SDK for Java_, see [Extensions]({{site.url}}{{site.baseurl}}/developer-documentation/extensions/).
```

Comment:
I believe we are looking at releasing extensions in 2.9 now

Reply:
@ryanbogan We're releasing extensions as an experimental feature in 2.8, correct? The header on the extensions page will say "experimental" so I think this can stay as is.

Reply:
No it got pushed back by one release